### ISSUE TITLE
Healpy and astropy clashing?

### ISSUE THREAD

**Issue by jleagle94** [healpy/healpy#643]
Hi all,

I am trying to run analyses using Fermipy/Fermitools and I keep getting an error that I have not run into before and I think it may be healpy and astropy clashing for some reason? Perhaps my Catalina is somehow clashing?

I have bogged my error down to `import healpy`
```
(fermipy2) jeagle-2:snr_g310.6-1.6 jordaneagle$ python
Python 2.7.15 | packaged by conda-forge | (default, Mar  5 2020, 14:58:04) 
[GCC Clang 9.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import healpy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jordaneagle/.local/lib/python2.7/site-packages/healpy/__init__.py", line 90, in <module>
    from .rotator import Rotator, vec2dir, dir2vec
  File "/Users/jordaneagle/.local/lib/python2.7/site-packages/healpy/rotator.py", line 38, in <module>
    representation_type="cartesian",
  File "/Users/jordaneagle/anaconda2/envs/fermipy2/lib/python2.7/site-packages/astropy/coordinates/sky_coordinate.py", line 208, in __init__
    kwargs = self._parse_inputs(args, kwargs)
  File "/Users/jordaneagle/anaconda2/envs/fermipy2/lib/python2.7/site-packages/astropy/coordinates/sky_coordinate.py", line 327, in _parse_inputs
    frame = valid_kwargs['frame'] = _get_frame(args, kwargs)
  File "/Users/jordaneagle/anaconda2/envs/fermipy2/lib/python2.7/site-packages/astropy/coordinates/sky_coordinate.py", line 1588, in _get_frame
    frame_cls = _get_frame_class(frame)
  File "/Users/jordaneagle/anaconda2/envs/fermipy2/lib/python2.7/site-packages/astropy/coordinates/sky_coordinate.py", line 1530, in _get_frame_class
    .format(frame, sorted(frame_names)))
ValueError: Coordinate frame barycentricmeanecliptic not in allowed values ['altaz', 'barycentrictrueecliptic', 'cirs', 'fk4', 'fk4noeterms', 'fk5', 'galactic', 'galacticlsr', 'galactocentric', 'gcrs', 'geocentrictrueecliptic', 'hcrs', 'heliocentrictrueecliptic', 'icrs', 'itrs', 'lsr', 'precessedgeocentric', 'supergalactic']

```
This small discussion https://gitmemory.com/issue/lwa-project/lsl/7/682213032 is the only thing I can find that reproduces my error. Help!

**Comment by zonca** [comment]
do you have the option of switching to Python 3? If not I would try to downgrade both `healpy` and `astropy` to older versions.

**Comment by mreineck** [comment]
Judging from the discussion you linked, this is most likely due to an outdated `astropy` on your system. Can you check the installed `astropy` version please?

Also, judging from the paths printed in your report, you still seems to be running Python 2, which is now well and truly beyond its end of life :-) You might want to consider switching to Python 3.

**Comment by mreineck** [comment]
Ooops, synchronized posting ;-)

**Comment by jleagle94** [comment]
Unfortunately, right now python 3 environment using fermipy is not available to me so I have to be on python 2.7 environment. The FermiPy/Fermitools only just recently (9/21/2020) updated their tools to work with python 3 and I have had no luck working on there yet. This should work fine as it did just a day ago in a python 2.7 environment but re-downloading is giving me these errors :(. 

here is the output for my healpy and astropy versions
```
(fermipy2) jeagle-2:snr_g310.6-1.6 jordaneagle$ pip show healpy
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Name: healpy
Version: 1.14.0
Summary: Healpix tools package for Python
Home-page: http://github.com/healpy
Author: C. Rosset, A. Zonca
Author-email: cyrille.rosset@apc.univ-paris-diderot.fr
License: GPLv2
Location: /Users/jordaneagle/.local/lib/python2.7/site-packages
Requires: scipy, numpy, astropy, six, matplotlib
Required-by: fermipy
(fermipy2) jeagle-2:snr_g310.6-1.6 jordaneagle$ pip show astropy
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Name: astropy
Version: 2.0.16
Summary: Community-developed python astronomy tools
Home-page: http://astropy.org
Author: The Astropy Developers
Author-email: astropy.team@gmail.com
License: BSD
Location: /Users/jordaneagle/anaconda2/envs/fermipy2/lib/python2.7/site-packages
Requires: pytest, numpy
Required-by: wcsaxes, healpy, regions, gammapy, fermipy
```

**Comment by jleagle94** [comment]
Hey all,

Just wanted to say I found the clash. It appears astropy versions > 2.0 clash with healpy 1.14 or less. downgrading healpy to 1.13.0 and astropy to 2.0.0 seemed to fix my issue.
